code/datums/station_traits/positive_traits.dm

```diff
@@ -184,7 +184,7 @@
 		/datum/job/hop = /obj/item/organ/internal/eyes/cybernetic/shield,
 		/datum/job/hos = /obj/item/organ/internal/cyberimp/brain/anti_stam, //not giving them thermals
 		/datum/job/janitor = /obj/item/organ/internal/cyberimp/eyes/hud/jani,
-		/datum/job/lawyer = /obj/item/organ/internal/heart/cybernetic/upgraded,
+		/datum/job/internal_affairs = /obj/item/organ/internal/heart/cybernetic/upgraded,
```

Comment:
This should be abbreviated like other multi-word titles in jobs
```suggestion
		/datum/job/iaa = /obj/item/organ/internal/heart/cybernetic/upgraded,
```

Reply:
All done.